### PR TITLE
Testing to see if master is correctly protected now

### PR DESCRIPTION
**Pull request by sbwilso7**

### CODE REVIEW

**Comment by sbwilso7** [comment]
Revisions to readme and testing pull request functionality

File: README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ First comment
 
 This is a change on newbranch
```
Comment (sbwilso7) [inline README.md:7]: great input

**Review by ser316asu — APPROVED** [review]
Approved, but I did not check if you did everything correctly.

**Review by ser316asu — APPROVED** [review]
Approved, but I did not check if you did everything correctly.

**Pull request merged**